public/external-text/default.yaml

```diff
@@ -181,6 +181,18 @@ help:
           Click on a column header to sort by that column.
 
           Click again to change the direction of sorting.
+          
+      - question: |
```

Comment:
As discussed in our all-hands meeting this AM, you may want to align the answer to this with [the documentation on the CMIP6 PDP landing page](https://www.pacificclimate.org/data/statistically-downscaled-climate-scenarios) (in particular, the paragraph headed with "B. Coupled Model Intercomparison Project Phase 6 (CMIP6)").